### DISCUSSION TITLE
How can I connect over SMB o SMB2 over meshcentral to other machine.

### DISCUSSION THREAD

**Discussion by tecnicosya** [Ylianst/MeshCentral#5817] [General]
Hi guys, I'm playing with this MeshCentral and wow so nice features.
I have installed the plugin called WorkFromHome is so nice.
In this way, is possible to use MeshCentral to connect to others machines for the Windows Shared Files?
I dont talk about the feature integrated to access to all files on HDD, I talk to use and explore the files with a simple link, and the user dont have access to meshcentral control panel, just the meshagent installed.
and if this will work on mobile devices.
I try WorkFromHome and not work from my android device, but is ok, need to know the limitations.

I ask this to make Mesh Way to use like a VPN.
But more simple if just have all in one Mesh server to connect all.

**Comment by dinger1986** [comment]
You can play with the mesh router, it's closer to a vpn, I don't know if that works with android however.

**Comment by tecnicosya** [comment]
> You can play with the mesh router, it's closer to a vpn, I don't know if that works with android however.

How can I activate the Mesh router?
I try with the pdf and talk about download a windows application, but the download link is down.
And My Meshcentral say need to activate Mesh Router. But how?

**Comment by silversword411** [comment]
https://ylianst.github.io/MeshCentral/meshrouter/

**Comment by tecnicosya** [comment]
> https://ylianst.github.io/MeshCentral/meshrouter/

Yes thank you. Today I see I will get it from my server.
But is more an administrative tool, not is transparent for the users and if I understand lost the routes after log off from Mesh Router

**Comment by dinger1986** [comment]
@si458 i don't think there's a way to do this
